src/utils/constants.ts

```diff
@@ -58,6 +58,15 @@ export const GWEI: BN = WEI.mul(new BN(1000000000))
 export const AVAXGWEI: BN = NANOAVAX.clone()
 export const AVAXSTAKECAP: BN = ONEAVAX.mul(new BN(3000000))
 
+export const KopernikusID = "1002"
```

Comment:
are you sure that those IDs are not defined somewhere else? quite strange, that there are still not there

Reply:
Nope, I didn't find it. They are not mentioned in README.md and examplesConfig.json by default uses `12345`